.changeset/lazy-plants-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+fix: unset cached resource for failed TX
```

Comment:
Shouldn't this be a breaking due to `set`'s new signature?

Reply:
@arboleya I did not specify this change because the `MemoryCache` ( renamed to `ResourceCache` ) was used only internally. Also, it is not exported by `fuels`.